fp/_baseConvert.js

```diff
@@ -279,6 +279,9 @@ function baseConvert(util, name, func, options) {
       setPlaceholder = true;
       func.placeholder = result.placeholder = placeholder;
     }
+    result.convert = function(options) {
+      return baseConvert(util, name, func, options);
```

Comment:
The `config` object in that scope has all it's [values set to true](https://github.com/lodash/lodash/blob/89f891ac86aa1b81a47598088c5188b4c358a4e6/fp/_baseConvert.js#L35-L40).

It seems that the mapping file is queried only when the actual conversion is being done. So `assing({}, config, options)` won't work here.